Server/middleware/auth.js

```diff
@@ -0,0 +1,7 @@
+async function authenticate(req, res, next) {
```

Comment:
Would you mind making this a stabby function - just for consistency as they're used everywhere else (not much reason for it.. other than I like stabby functions)